topics/ecology/tutorials/Biodiversity data exploration/tutorial.md

```diff
@@ -285,3 +266,22 @@ Final absence correlation plot:
 
 Here, you just explored your biodiversity dataframe properly and you know a lot more about your data. You can now peacefully make your statiscal analyses as most of the red flags you can get have been revealed by this toolsuite ! Enjoy !
 
+# Want to spatially anoymize your data?
+
+A first step of this tutorial will show you how you can simply apply spatial coordinates anonymization if you want to share data without spatial context, particularly of interest if you want to share endangered species oriented data.
+
+# Bonus!
+## Spatial coordinates anonymization
```

Comment:
```suggestion
## Bonus! Spatial coordinates anonymization
```

Reply:
(just to try and keep the heading levels hierarchical.)

Reply:
THANK YOU Helena! Hope all is ok on your side!!!